BUILD_IMAGE_VERSION

```diff
@@ -1 +1 @@
-stackrox-build-0.3.55
```

Comment:
Has the issue at the origin of the revert https://github.com/stackrox/stackrox/pull/4768 been understood and resolved ?

Reply:
`-dirty` was added because `generated/` files were changed